### PR TITLE
Zapped `using namespace std;` everywhere

### PR DESCRIPTION
**Pull request by hannes**

### CODE REVIEW

File: src/function/scalar/nested/struct_pack.cpp [hunk 1]
```diff
@@ -34,7 +32,7 @@ static void struct_pack_fun(DataChunk &args, ExpressionState &state, Vector &res
 
 static unique_ptr<FunctionData> struct_pack_bind(ClientContext &context, ScalarFunction &bound_function,
                                                  vector<unique_ptr<Expression>> &arguments) {
-	set<string> name_collision_set;
+	std::set<string> name_collision_set;
```
Comment (Mytherin) [inline src/function/scalar/nested/struct_pack.cpp:35]: Can this be turned into an unordered set, or is set required here? (I believe `set` is implemented as a tree and `unordered_set` as a HT - not very significant but unordered_set is generally a bit more efficient if you only need to do point inserts/queries, and we use it "by default" in most places so it's a bit more consistent).

**Comment by Mytherin** [comment]
Excellent! This has been on the reformat goals for some time :)

In the few places that we now have to use `std::`, perhaps we could add headers similar to e.g. our current [string.hpp](https://github.com/cwida/duckdb/blob/master/src/include/duckdb/common/string.hpp) that include a part of the standard library and expose that in the `duckdb` namespace? For example, we could add `using std::to_string` to that header, and we could add a `duckdb/common/random.hpp` that includes <random> and uses the random devices, uniform distribution, etc that we use.

File: src/function/scalar/nested/struct_pack.cpp (hunk 1 above)
Comment (hannesmuehleisen) [reply src/function/scalar/nested/struct_pack.cpp:35]: sure did not do so much semantic analysis

**Comment by hannesmuehleisen** [comment]
> In the few places that we now have to use `std::`, perhaps we could add headers similar to e.g. our current [string.hpp](https://github.com/cwida/duckdb/blob/master/src/include/duckdb/common/string.hpp) that include a part of the standard library and expose that in the `duckdb` namespace? For example, we could add `using std::to_string` to that header, and we could add a `duckdb/common/random.hpp` that includes and uses the random devices, uniform distribution, etc that we use.

I think it makes most sense for stuff that's used a lot, indeed `to_string` is the only thing I can think of. The random stuff is pretty localised and can just be `used there no?

**Comment by Mytherin** [comment]
Sure, that makes sense. Perhaps `pair` and `make_pair` as well, but otherwise we mostly already have covered the parts we use a lot.

**Pull request merged**